google/cloud/bigtable/internal/bulk_mutator.cc

```diff
@@ -127,7 +127,7 @@ std::vector<int> BulkMutatorState::OnRead(
   return res;
 }
 
-void BulkMutatorState::OnFinish(google::cloud::Status finish_status) {
+void BulkMutatorState::OnFinish(google::cloud::Status const& finish_status) {
```

Comment:
This change makes it impossible for callers to transfer `finish_status` when they have no more use for it, forcing an additional copy that may be unneeded.

In addition, this change seems unrelated: you could do it in a separate PR.  Future you would appreciate if each PR is about one thing only.